__tests__/api.test.js

```diff
@@ -169,6 +169,34 @@ describe('GET/api/articles(sort queries)',()=>{
       });
   });
 })
+describe('GET/api/articles(topic query)',()=>{
+  it("status 200: responds with articles of given topic", () => {
+    return request(app)
+      .get("/api/articles?topic=cats")
+      .expect(200)
+      .then(({ body }) => {
+        body.articles.forEach((item) => {
+          expect(item.topic).toBe("cats");
+        });
```

Comment:
Very good, you have already got tests checking what articles look like so no point repeating yourself- you should only check the topic here, nice!